src/async_impl/client.rs

```diff
@@ -874,3 +872,18 @@ fn make_referer(next: &Url, previous: &Url) -> Option<HeaderValue> {
     referer.set_fragment(None);
     referer.as_str().parse().ok()
 }
+
+fn attach_header(headers: &mut HeaderMap, cookie_store: &cookie::CookieStore, url: &Url) {
```

Comment:
My only thought is that inside this file, `attach_header` may be a bit of a vague name. How about `add_cookie_header`?

Reply:
aggred with the `add_cookie_header` :)